app/models/post.rb

```diff
@@ -0,0 +1,13 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: :author_id_id
```

Comment:
- [ ] Great job so far! 👏 You're on the right track with setting up the associations in your models. However, there are a couple of improvements and corrections needed. Let's dive into the review:

   - In Rails, when you define associations like `has_many` or `belongs_to`, Rails automatically assumes the foreign key based on naming conventions unless you explicitly specify it. So, you don't need to add `_id` in the foreign key names when using the typical Rails naming conventions.

**Instead of 👇👇**
```
   belongs_to :author, class_name: 'User', foreign_key: :author_id_id
```

**Do this 👇👇**
```
 belongs_to :author, class_name: 'User', foreign_key: :author_id
```

Happy coding!